README.md

```diff
@@ -487,6 +508,9 @@ if (result.error) throw result.error;
     sourceMap: {
         // source map options
     },
+    nameCache: {
+        // name cache object
+    },
```

Comment:
Done in https://github.com/mishoo/UglifyJS2/pull/2176/commits/e63b9ab27e6f956232c21d10506f4bd5e7c35524